constructr-machine/src/main/scala/de/heikoseeberger/constructr/machine/ConstructrMachine.scala

```diff
@@ -147,7 +147,11 @@ final class ConstructrMachine[A: Coordination.AddressSerialization, B <: Coordin
   }
 
   when(State.AddingSelf, coordinationTimeout) {
-    case Event(Coordination.SelfAdded(_), _) => goto(State.RefreshScheduled).using(stateData.copy(coordinationRetriesLeft = coordinationRetries))
+    case Event(Coordination.SelfAdded(context: B#Context), _) =>
```

Comment:
Add an `@unchecked` annotation, like above in line 113 `case Event(nodes: List[A] @unchecked, _) => ...`.